frontend/server/src/main/java/org/pytorch/serve/http/ManagementRequestHandler.java

```diff
@@ -329,8 +337,19 @@ private void updateModelWorkers(
                                     modelManager.scaleRequestStatus(modelName, modelVersion);
                             if (HttpResponseStatus.OK.equals(v)) {
                                 if (status) {
-                                    NettyUtils.sendJsonResponse(
-                                            ctx, new StatusResponse("Workers scaled"), v);
+                                    String msg =
```

Comment:
I am curious to understand if this message needs modification. Can't this be a generic message of how many workers are we scaling this to? If we elaborate on this message, then customers who maintain logs will know that `POST /models`, which is register model,  was updated with `n` number of workers. Adding a flag to determine the code path, ie., if this is coming from register-model or from scale model, seems a little hacky.

Reply:
The idea is to keep the response message consistent for registration request with/without initial workers. I agree its a bit hacky.

Reply:
Keeping these changes as message give clear information.

Reply:
@vdantu Let me know if this LGTY

Reply:
I am still not convinced that passing a boolean flag to determine the error message is the the only option. But, this is is **not** a blocker for the PR. We can look into how to refactor this code in the future (may be return the `future` to the method above and populate the error message there or some such).